prepare.ps1

```diff
@@ -1,15 +1,6 @@
-# Configuring
-Set-ExecutionPolicy -ExecutionPolicy Bypass -Force
-
 Import-Module BitsTransfer
 [Net.ServicePointManager]::SecurityProtocol = "tls12, tls11, tls" #Convince Powershell to talk to sites with different versions of TLS
 
-# Installing Chocolatey 
-Invoke-Expression ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
if you want to test this bit and it works happy for you to keep it in :)

Reply:
Hi,

I have tested on my notebook (uninstalling chocolatey first). But i'll do some more tests and ask friends to do that too. By now, I moved this feature to another branch and I intend to submit a new PR if I have sucess on them.

That being said, if you want to accept PR as it is now (without the auto-installer for choco), I think it's safe

Reply:
hey when you're ready to submit the other PR i'll test that. bit busy with work atm. Thanks heaps PRElias!